tests/datasource/fluent/data_asset/data_connector/test_dbfs_data_connector.py

```diff
@@ -39,11 +38,6 @@ def test__get_full_file_path_pandas(fs: FakeFilesystem):
         module_dir = pathlib.Path(module.__file__).parent
         fs.add_real_directory(module_dir, lazy_read=False)
 
-    # Copy google credentials into fake filesystem if they exist on your filesystem
```

Comment:
I don't understand why this was here, but it was causing CI failures now that I'm adding that to the marker tests. https://github.com/great-expectations/great_expectations/actions/runs/11861098769/job/33057909802